File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/strategy/transport/Car.java

package com.bigboxcode.designpattern.strategy.transport;

public class Car implements Transport {
    @Override
    public void start() {
        System.out.println("Car started");
    }

    @Override
    public void stop() {
        System.out.println("Car stopped");
    }

    @Override
    public void repair() {
        System.out.println("Car repair");
    }

    @Override
    public void getInfo() {
        System.out.println("Transport type: Car");
        System.out.println("Number of wheels: 4");
        System.out.println("Average Weight: 4,000 Pounds");
    }

    @Override
    public void operate() {
        System.out.println("Driving car ............");
    }
}

File: design-patterns/php/strategy/transport/Car.php
<?php
// Car.php

namespace BigBoxCode\DesignPattern\Strategy\Transport;


class Car implements Transport {
    public function start(): void {
        echo "Car started\n";
    }

    public function stop(): void {
        echo "Car stopped\n";
    }

    public function repair(): void {
        echo "Car repair\n";
    }

    public function getInfo(): void {
        echo "Transport type: Car\n";
        echo "Number of wheels: 4\n";
        echo "Average Weight: 4,000 Pounds\n";
    }

    public function operate(): void {
        echo "Driving car ............\n";
    }
}